lib/expectations_runner.rb

```diff
@@ -19,9 +25,18 @@ class Mumukit::Inspection::PlainInspection
 
   def expectations
     {
-      'HasWhile' => /AST\(while/
+      'HasWhile' => use(/AST\(while/),
```

Comment:
I'd love to skip the parentheses here, but I didn't know how :disappointed: 
Any ideas?